src/ui/render_config.rs

```diff
@@ -127,6 +127,12 @@ pub struct RenderConfig {
     /// a separator from the prefix.
     pub option: StyleSheet,
 
+    /// Style sheet for the option that is currently selected.
+    ///
+    /// Note: a non-styled space character is added before the option value as
+    /// a separator from the prefix.
+    pub selected_option: StyleSheet,
```

Comment:
I'd like your opinion on that:

Do you think it would make sense for selected_option to be an Option<StyleSheet> instead? In a sense that a None would mean the style falls back to option and Some(empty()) means that the selected option is intentionally not styled?

I think it can be a "nice to have", and it is easy to implement. But would people really use it this way and do we want to add this additional indirection?

Lmk what you think

Reply:
Since current implementation does not support a case where user wants `selected_option` to be `StyleSheet::empty() `and `option` something else, I think this is a nice feature. 

On the other hand I'm a bit worried it might cause confusion between None and empty style, but that can be solved through docs. 

Not sure if there is some other way to offer a user to chose from no styling, falling back to `option` and custom style. Currently I'm fine with Option implementation

Reply:
Yep let's go with Option then

Reply:
Done in https://github.com/mikaelmello/inquire/pull/88/commits/c701464b25e398d3b748c15e84ba5e1dc9b8ddbd